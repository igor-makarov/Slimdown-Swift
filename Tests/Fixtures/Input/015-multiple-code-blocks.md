One

```
function foo () {
	print ('Foo');
}
```

Two

```
function foo () {
	print ('Foo');
}
```

Three