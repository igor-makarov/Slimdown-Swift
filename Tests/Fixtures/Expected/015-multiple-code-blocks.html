<p>One</p>

<pre><code>function foo () {<br>	print ('Foo');<br>}</code></pre>

<p>Two</p>

<pre><code>function foo () {<br>	print ('Foo');<br>}</code></pre>

<p>Three</p>